pylint/testutils/lint_module_test.py

```diff
@@ -175,6 +178,7 @@ def _get_actual(self):
         return received_msgs, received_output_lines
 
     def _runTest(self):
+        __tracebackhide__ = True  # pylint: disable=unused-variable
```

Comment:
Not related to this MR but it look like a false positive. It's used somewhere just not in this class.

Reply:
It's only used by `pytest` during the actual run, so technically it's unused in the code itself.

Reply:
We're going to get showered by issue about this 😨

Reply:
You have to explain that 😅

Reply:
> We're going to get showered by issue about this 😨

@Pierre-Sassoulas Sorry to ping you about that, but I'm just curious what you meant by it.

Reply:
Ho sorry, I misunderstood your mis understanding :D I think we're going to have a lot of false positive issue opened about this. `__dunder__` are valids and often used. pytest is used a lot too.

Reply:
Not sure this is really an issue. At least it's the first time I've seen it and in the context completely understandable. The only to fix it would probably be to add an exclude option

Reply:
We could exclude dunders from the check though ? Because they're not really private.